packages/mobile/src/in-house-liquidity/index.ts

```diff
@@ -40,13 +40,18 @@ const getSerializedSignature = async (
   const signature = signMessage(message, privateKey, accountMTWAddress)
   return serializeSignature(signature)
 }
-const getAuthHeader = async (
+
+const getAuthAndDateHeaders = async (
```

Comment:
sorry to just bring this up now, but this could probly use a docstring explaining how to use it

Reply:
the fact that you need an HTTP verb prefix, in particular, is not intuitive